### ISSUE TITLE
[Research] A11y: Error with Storybook iframe.html must have a main landmark

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#3253]
**User Story:**
As Spark we want Storybook iframe.html to contain a `<main>` element.

**Current State:**
Axe error: Document must have one main landmark
 
Storybook iframe.html don't contain a `<main>` element. This should be added as a wrapper to the page or in every story where we have sprk-o-Box.

**notes:**
SME: Bob/Olya
Possible solution? 
- Storybook iframe.html will contain a `<main>` element.
- iframe.html `<main>` element will be added as a wrapper to the page 
- iframe.html `<main>` element will be added as a wrapper everywhere we have sprk-o-Box

**AC:**
- Why are we getting the error? how would we stop this from happening?

![image](https://user-images.githubusercontent.com/46761151/88559434-d1fcca00-cffa-11ea-98e4-8d0f67c3436b.png)

**Comment by bobwise** [comment]
from Olya:

https://dequeuniversity.com/rules/axe/3.5/landmark-one-main?application=AxeFirefox

**Comment by afebbraro** [comment]
Supplying this for the research task. Doesn't look like we are supposed to have main landmarks on every single iFrame. See this https://github.com/dequelabs/axe-core/pull/418#issuecomment-322880375.

**Comment by RVMendoza** [comment]
# **➡️ After research, my conclusion is that we keep things as is and not change anything. Ignore these errors when evaluating an an isolated iframe.**

## Two Main Reasons
1. Our use of the axe tool is to help us improve user experience and accessibility. This fix is only for a specific use case and doesn’t ultimately improve or give us intel about the component. 
    - This axe evaluation would be useful in context of a page, but since we’re in the business of making components, it doesn’t apply.
    - This applies to the “Document must contain a level-one heading” issue it finds as well.
    -  This is akin to our conversations about writing tests. Our goal is not to write tests to to get 100%, but it's to test its important functionalities. The latter is about getting an arbitrary "A", and the former is about creating robust components.

2. Informed by @Afebrarro’s comment, this could introduce a problem for us in our Gatsby page. We would be getting rid of a problem (which we learn isn’t one from reason 1) and creating new ones.
    - We use the gatsby site to show multiple iframes housing each variant of a component. If we were to do this fix, each iframe would have its own `main` and could cause confusing axe reports.
    - Gatsby pages are more public than the three steps it takes to recreate the scenario we’re trying to fix. It doesn’t seem worth it.


Willing to have further conversation, of course!
**My recommendation is to close this issue and ignore page-context related axe issues and focus on component related accessibility problems going forward.**
